genesyscloud/telephony_providers_edges_phonebasesettings/resource_genesyscloud_telephony_providers_edges_phonebasesettings.go

```diff
@@ -118,6 +121,9 @@ func updatePhoneBaseSettings(ctx context.Context, d *schema.ResourceData, meta i
 			State:        (*phoneBaseSettings.Lines)[0].State,
 		},
 	}
+	if lineProperties := BuildTelephonyLineBaseProperties(d); lineProperties != nil {
+		(*phoneBase.Lines)[0].Properties = lineProperties
+	}
```

Comment:
We should be careful here to not overwrite all the data in Properties that is generated by the API after creation. The team that owns the API might be able to advise more on this, but I think it would be better to read the current state of `Properties` for this line and update just the bool and int field.

Reply:
for the properties attributes across telephony, the backend only mutates the fields that we send, the default fields are not overwritten.